utility/firebase.js

```diff
@@ -199,43 +199,33 @@ const fireDb = {
       SignUpText: signupText || ''
     });
   },
-  addSponsorInformation: async (website, sponsor) => {
+  setSponsor: async (website, sponsor) => {
     const ref = db
-      .collection(webCollection)
+      .collection("Hackathons")
       .doc(website)
-      .collection('Sponsors');
-    await ref.add({
-      image: sponsor.image,
+      .collection('Sponsors')
+      .doc(sponsor.name);
+    await ref.set({
+      imgURL: sponsor.image,
       name: sponsor.name,
-      url: sponsor.url,
-      rank: sponsor.rank,
-      altImage: sponsor.altImage
-    });
+      link: sponsor.link,
+      tier: sponsor.tier,
+      lastmod: sponsor.lastmod
+    })
   },
-  async deleteSponsor(website, image) {
-    let altImage = false;
-    try {
-      const sponsors = await db
-        .collection(webCollection)
-        .doc(website)
-        .collection('Sponsors')
-        .get();
-      sponsors.forEach(async sponsor => {
-        if (sponsor.data().image === image) {
-          altImage = !!sponsor.data().altImage;
-          await sponsor.ref.delete();
-        }
-      });
-    } catch (e) {
-      return false;
-    }
-    const ref = storage.ref(`${website}/${image}`);
+  deleteSponsor: async(website, sponsorName) => {
+    const ref = db
+      .collection("Hackathons")
+      .doc(website)
+      .collection("Sponsors")
+      .doc(sponsorName);
```

Comment:
This delete function won't work properly because sponsors require an image upload (we host the images). I'm not sure if we're changing this but if not then we'll have to handle deleting the images from storage as well (like the original function did).